### ISSUE TITLE
`flyctl ips private` doesn't support `--json` flag

### ISSUE THREAD

**Issue by jvns** [superfly/flyctl#364]
The `flyctl ips list` command supports JSON, but `flyctl ips private` doesn't: when I pass `--json` it gives me a table instead of JSON.

```
$ flyctl --json  ips  private
ID       REGION IP                             
df6f8339 yyz    <the instance IP>
```

**Comment by michaeldwan** [comment]
@jvns good catch, we'll get that fixed in the next update

**Comment by michaeldwan** [comment]
@jvns this is fixed in the latest update (v0.0.175)
